### PR TITLE
Update pt_BR translation

### PR DESCRIPTION
**Pull request by tiagoFlach**
Complete translations to Brazilian Portuguese

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -256,6 +258,7 @@ msgstr "Copyright 2021 Dave Patrick"
 #: src/main.py:79
 msgid "translator-credits"
 msgstr "Vitor Cardoso"
+msgstr "Tiago Lucas Flach"
```
Comment (SeaDve) [inline po/pt_BR.po:261]: I think you don't need to put msgstr here on this line

**Comment by SeaDve** [comment]
Thanks very much!

**Review by SeaDve — COMMENTED** [review]
Would you mind if i push to your branch?

**Comment by tiagoFlach** [comment]
Do what you want

**Comment by tiagoFlach** [comment]
👍👏

**Pull request merged**